build/vsts-ci.yml

```diff
@@ -58,7 +58,54 @@ phases:
       artifactType: container
 
 ################################################################################
-- phase: Windows
+- phase: Windows_x86
+################################################################################
+  variables:
+    BuildConfig: Release
+    OfficialBuildId: $(BUILD.BUILDNUMBER)
+    DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+    DOTNET_MULTILEVEL_LOOKUP: 0
+    _SignType: real
+    _UseEsrpSigning: true
+    _TeamName: DotNetCore
+  queue:
+    name: DotNetCore-Build
+    demands: 
+      - agent.os -equals Windows_NT
+  steps:
+
+  - task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
+    displayName: Install MicroBuild Signing Plugin
+    inputs:
+      signType: '$(_SignType)'
+      zipSources: false
+      esrpSigning: '$(_UseEsrpSigning)'
+    env:
+      TeamName: $(_TeamName)
+    continueOnError: false
+    condition: and(succeeded(), in(variables._SignType, 'real', 'test'))
+
+  - script: ./build.cmd -buildNative -$(BuildConfig) -buildArch=x86
```

Comment:
Do we need `-buildNative`? We aren't specifying it for the x64 build...

Reply:
Yes, see [here](https://github.com/dotnet/machinelearning/pull/1306#discussion_r226813126) for an explanation.

Reply:
Does the yml file support comments? Might be a useful annotation.